front/lib/api/assistant/conversation.ts

```diff
@@ -1745,9 +1734,7 @@ async function* streamRunAgentEvents(
         runIds.push(event.runId);
         await agentMessageRow.update({
           runIds,
-          content: event.text,
```

Comment:
So right now the only reason we keep `agent_generation_success`  as an event is that we need `runId`. I think we could decide to have the run ID in `AgentMessageContent` event and use that one instead.  cc @spolu

Reply:
I think it's good to have that event esp if it has a purpose. It makes a more comprehensible API for our users also IMHO.

Reply:
> I think it's good to have that event esp if it has a purpose

But it doesn't have a purpose really. This event is always immediately followed by `agent_message_success`, which contain all agent message info. 
Less events make the API simpler.

Reply:
(With the PR I am proposing this event is **no longer** exposed to end users)